samples/azure-quantum/chemistry/data/broombridge/caffeine.yaml

```diff
@@ -0,0 +1,92 @@
+$schema: https://raw.githubusercontent.com/Microsoft/Quantum/master/Chemistry/Schema/broombridge-0.1.schema.json
```

Comment:
Likely, but by the same token as we need copyright headers on individual source files, it may be a good practice to make sure there's a header on these sources as well.

Reply:
I didn't see any copyright headers on the Broombridge files in the other samples either so I am not sure if we should add them here: https://github.com/microsoft/Quantum/tree/main/samples/chemistry/IntegralData/Broombridge_v0.2